x-pack/plugins/rule_registry/server/rule_data_client/rule_data_client.ts

```diff
@@ -215,10 +216,8 @@ export class RuleDataClient implements IRuleDataClient {
           } else {
             this.options.logger.debug(`Writing is disabled, bulk() will not write any data.`);
```

Comment:
Is this a possible path? Isn't `this.clusterClient` always set if we successfully initialized the writer?

Reply:
It should be yes! But I don't think it hurts to check and it make Typescript happy.